waiter/src/waiter/cookie_support.clj

```diff
@@ -78,6 +79,12 @@
               (assoc-in response [:headers "set-cookie"] new-header)))]
     (ru/update-response response add-cookie-into-response)))
 
+(defn add-encoded-cookie
+  "Inserts the provided name-value pair as a Set-Cookie header in the response after encoding the value."
+  [response password name value age-in-seconds http-only?]
```

Comment:
Not sure if this answers your question, but the `x-auth-expires-at` does not have `HttpOnly=true` as we want Javascript code to be able to view the cookie.

Reply:
I indeed saw that.  `add-cookie`, not `add-encoded-cookie`, is used for `x-auth-expires-at`.  I'd like to avoid implying that  exposing non `http-only?`, encrypted cookies is an OK thing to do unless we have a compelling use case.